lib/v2/screens/transfer/send_search_user/send_search_user_screen.dart

```diff
@@ -1,18 +1,20 @@
 import 'package:flutter/material.dart';
 import 'package:seeds/v2/components/search_user/search_user_widget.dart';
 import 'package:seeds/v2/datasource/remote/model/member_model.dart';
+import 'package:seeds/providers/services/navigation_service.dart';
 
 /// SendSearchUserScreen SCREEN
 class SendSearchUserScreen extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
+    Future<void> onResult(MemberModel selectedUser) async {
+      print("onResult:" + selectedUser.account);
+      await NavigationService.of(context).navigateTo(Routes.sendEnterData, selectedUser);
+    }
+
     return Scaffold(
       appBar: AppBar(title: const Text("Send")),
       body: Container(padding: const EdgeInsets.all(16), child: SearchUserWidget(resultCallBack: onResult)),
```

Comment:
I see this SearchUserWidget is a global component so elsewhere we are looking for users? 
If so then I understand the reason for the callback that returns the selected user 👍